pkg/cli/updater/resolver/resolver.go

```diff
@@ -73,6 +87,44 @@ func (v *Version) String() string {
 	return v.Tag
 }
 
+// NewVersion creates a new version from a tag and commit.
+func NewVersion(tag, hash string) (*Version, error) {
```

Comment:
This isn't really related to this PR, but it allowed me to cleanup some code duplication.